### PR TITLE
Cargo - Add documentation on adding cargo via config

### PR DESCRIPTION
**Pull request by johnb432**
**When merged this pull request will:**
- Title.
- Minor cleanup.

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/cargo/CfgVehicles.hpp [hunk 1]
```diff
@@ -48,15 +48,6 @@ class CfgVehicles {
     class Car: LandVehicle {
         GVAR(space) = 4;
         GVAR(hasCargo) = 1;
-        class ACE_Cargo {
```
Comment (johnb432) [inline addons/cargo/CfgVehicles.hpp:51]: I'm actually not too certain about this change. Could this cause a BWC issue?

**Comment by Drofseh** [comment]
I'm not sure it's a good idea to reduce the avaiable cargo space of a vehicle this late in the game. Likewise for increasing the cargo size of vehicle.
It could screw up existing mission by preventing their existing cargo from fitting.

**Comment by johnb432** [comment]
> It could screw up existing mission by preventing their cargo from fitting.

Whilst I do agree, being able to fit a same sized object within an object should not be allowed. Imo the latter outweighs the prior.

**Comment by johnb432** [comment]
More people in were favor of not changing sizes, so PR has been adapted accordingly.

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**